nHentaiDownloader.py

```diff
@@ -39,27 +38,27 @@ def printProgressBar (iteration, total, prefix = '', suffix = '', decimals = 1,
     soup = BeautifulSoup(response.text, 'html.parser')
     comicTitle = soup.find('h2', class_ = 'title')
     comicTitle = comicTitle.find('span', class_ = 'pretty').getText()
-    
+
     print('Title: ' + comicTitle)
     imggalleryURL = soup.find_all('a', class_='gallerythumb')
     page = len(imggalleryURL)
     imggalleryURL = imggalleryURL[0].find('img').get('data-src')[:-6]
     imggalleryURL = imggalleryURL[imggalleryURL.find('.'):]
-    
+
     try:
         os.mkdir(comicTitle)
     except:
         pass
-        
+
     print('Processing...')
     printProgressBar(0, page, prefix = 'Progress:', suffix = 'Complete', length = 50)
     
     for i in range(page + 1):
-        imgURL = "https://i" + imggalleryURL + str(i) + ".jpg"
+        imgURL = f"https://i{imggalleryURL}{str(i)}.jpg"
 
         response = requests.get(imgURL)
         if response.status_code == 200:
-            with open(".\\"+ comicTitle + "\\"+ str(i) + ".jpg", 'wb') as f:
+            with open(f"{os.getcwd()}/{comicTitle}/.\\{comicTitle}\\{str(i)}.jpg", 'wb') as f:

```

Comment:
FileNotFoundError 
Maybe change it to (f"{os.getcwd()}/{comicTitle}/{str(i)}.jpg) will better

Reply:
sorry but it worked in ubuntu so sorry